CHANGELOG.md

```diff
@@ -2,6 +2,69 @@
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.104.0](https://github.com/returntocorp/semgrep/releases/tag/v0.104.0) - 2022-07-13
+
+### Added
+
+- Added a new feature that was described in CLI-123. This was done support our overarching goal to improve the tool. (cli-123)
```

Comment:
Well this example entry should not have ended up here

Reply:
Do you mean that it should not have been included by towncrier, or that the entry should not have been written by the person who wrote it?

Reply:
Towncrier should not have included it - not sure why it was included.